Parser/Test updates, various updates

jaydiablo opened the pull request:
Some fixes to a couple of new parsers, pretty print JSON from JS parsers.

Also adding some missing top-1000-* files and some php-cs-fixer updates.

Removing phpstan ignore as well (after fixing the error).

jaydiablo committed:
Parser/Test updates, some fixes to a couple of new parsers, pretty print JSON from JS parsers

Also adding some missing top-1000-* files and some php-cs-fixer updates.

Removing phpstan ignore as well (after fixing the error).

jaydiablo committed:
additional php-cs-fixer fix

mimmi20 commented on parsers/whichbrowser-php-2/scripts/parse.php:
```diff
@@ -58,7 +58,7 @@
                 'brand'    => !empty($r->device->manufacturer) ? $r->device->manufacturer : '',
                 'type'     => !empty($r->device->type) ? $r->device->type : '',
-                'ismobile' => $r->isMobile() ? true : false,
+                'ismobile' => $r->isMobile() ? 'true' : 'false',
```
Do all parsers return these values as strings?

jaydiablo replied:
Hmm, no, but in this case I was seeing differences between this parser and the whichbrowser test suite because the test suite was expecting a string.

I'll look over all parsers and tests and make sure they're all using one or the other (whichever the majority is at this point).

jaydiablo committed:
making sure all ismobile properties in tests and parsers is bool or null

Ran the whichbrowser parsers against the whichbrower test suite and confirmed that they didn't have a difference for the ismobile property.

jaydiablo committed:
more updates, slight scoring change, updated README

mimmi20 commented on parsers/3rd-eden-useragent-js-2/scripts/parse.js:
```diff
@@ -45,23 +45,32 @@ lineReader.on('line', function (line) {
             'browser': {
                 'name': r.family,
-                'version': r.toVersion()
+                'version': r.toVersion() == '0.0.0' ? '' : r.toVersion()
```
please compare with `===`

jaydiablo committed:
adding prettier, removing yoda style, more divide by zero protection

jaydiablo committed:
removing argument forwarding from yarn command, not needed in yarn 1.x
